agents/chwp/hwpbbb_agent.py

```diff
@@ -529,6 +529,12 @@ def start_acq(self, session, params):
 
             self.take_data = True
 
+            # Prepare data_cache for session.data
+            self.hwp_freq = None
```

Comment:
Makes sense, but can I initialize by `self.hwp_freq = -1` instead?
If encoder is accidentally disconnected, one of the power supply of chwp electronics is accidentally broken or network is down, it results in encoder is reading no value and showing 0 Hz but hwp is still rotating.

For safety purposes, I prefer to initialize by -1. This mean encoder has never read any value since agent is up.
The normal `self.hwp_freq` is non negative.